test/test_line.py

```diff
@@ -285,6 +285,13 @@ def test_attrib_b(self):
         with self.assertRaises(AttributeError):
             del line.b
 
+    def test_attrib_length(self):
+        """a full test for the length attribute"""
+        expected_length = 3.0
+        line = Line(1, 4, 4, 4)
+
+        self.assertEqual(line.length, expected_length)
```

Comment:
Could you add more tests? like make two different lines with the same length and check if the two are the same, measure a line, move it and remeasure it to check if the length is correctly updated etc.